package Exception_Handling;

public class Try_with_Recource {
  public static void main(String[] args) {
	
}
}
